core/src/main/java/com/nextdoor/bender/config/BenderConfig.java

```diff
@@ -118,15 +118,7 @@ protected BenderConfig() {
 
     private static final ArrayList<Class> abstractConfigClasses = new ArrayList<Class>() {
       {
-        add(HandlerConfig.class);
```

Comment:
Is this change related?

Reply:
It’s tangentially related. Right now config Classes are discovered by looking for subclasses for particular classes like ‘DeserializerConfig.class’. However if one chooses to add a config to a config then it is unable to be discovered. This is the case for the ‘SubstitutionConfig’ the objects in it for ‘SubSpec*Config’. This change allows scanning for all ‘AbstractConfig’ classes which would include any configs implemented that aren’t an operation or deserializer.